bin/proving-service/src/commands/update_workers.rs

```diff
@@ -68,13 +68,13 @@ impl UpdateWorkers {
         let proxy_config = ProxyConfig::load_config_from_file()?;
 
         // Create the full URL
-        let url = format!("http://{}:{}?{}", proxy_config.host, proxy_config.port, query_params);
+        let url = format!(
+            "http://{}:{}?{}",
+            proxy_config.host, proxy_config.workers_update_port, query_params
+        );
 
         // Create an HTTP/2 client
-        let client = Client::builder()
-            .http2_prior_knowledge()
-            .build()
-            .map_err(|err| err.to_string())?;
+        let client = Client::builder().http1_only().build().map_err(|err| err.to_string())?;
```

Comment:
The comment says "HTTP/2" but seems like we are creating an `http1` client? Also, is there a reason to use HTTP/1 here specifically?

Reply:
There isn't a particular reason for using http/1.x. Can be replaced with http2